backend/src/models/Post.js

```diff
@@ -39,5 +39,6 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  imageBlurred: { type: 'boolean', default: false },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T11:57:28Z" title="Friday, December 6th 2019, 12:57:28 pm +01:00">Dec 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -39,5 +39,6 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  blurImage: { type: 'boolean', default: false },
```


```suggestion
  imageBlurred: { type: 'boolean', default: false },
  imageBlurredByModerator: { type: 'boolean', default: false },
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T11:58:04Z" title="Friday, December 6th 2019, 12:58:04 pm +01:00">Dec 6, 2019</time>_

---


talking about it, it would be nice to keep the image properties in the database start with `image`

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-10T10:33:25Z" title="Tuesday, December 10th 2019, 11:33:25 am +01:00">Dec 10, 2019</time>_

---


ok

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-12T09:44:49Z" title="Thursday, December 12th 2019, 10:44:49 am +01:00">Dec 12, 2019</time>_

---


I changed the database name from "blurImage" to "imageBlurred".